### PR TITLE
tests: use apiBaseSuite for snapshots tests, fix import endpoint path

### PR DESCRIPTION
**Pull request by stolowski**
Import tests use wrong endpoint path (which isn't relevant for the test because it calls api command directly but is confusing). This PR fixes the path and also updates the test to use apiBaseTest (and with this, the endpoint path actually needs to be correct).

### CODE REVIEW

File: daemon/api_snapshots_test.go [hunk 1]
```diff
@@ -34,38 +34,21 @@ import (
...
-	snapstate.ReplaceStore(st, storetest.Store{})
-	dirs.SetRootDir(c.MkDir())
-}
-
-func (s *snapshotSuite) TearDownTest(c *check.C) {
-	s.o = nil
-	s.d = nil
+	s.apiBaseSuite.SetUpTest(c)
+	d := s.daemonWithStore(c, s)
+	d.Overlord().Loop()
```
Comment (pedronis) [inline daemon/api_snapshots_test.go:50]: do we need the Loop actually?

File: daemon/api_snapshots_test.go [hunk 2]
```diff
@@ -34,38 +34,21 @@ import (
...
-	defer st.Unlock()
-	snapstate.ReplaceStore(st, storetest.Store{})
-	dirs.SetRootDir(c.MkDir())
-}
-
-func (s *snapshotSuite) TearDownTest(c *check.C) {
-	s.o = nil
-	s.d = nil
+	s.apiBaseSuite.SetUpTest(c)
+	d := s.daemonWithStore(c, s)
```
Comment (pedronis) [inline daemon/api_snapshots_test.go:49]: looking at the old code it seems we might need a slightly different variant of this that takes an Overlord instead? I haven't needed it yet, but doesn't mean we can't add it

File: daemon/api_snapshots_test.go (hunk 1 above)
Comment (stolowski) [reply daemon/api_snapshots_test.go]: Indeed not needed, removed.

**Review by pedronis — APPROVED** [review]
thank you

**Review by mvo5 — APPROVED** [review]
Thank you!

**Pull request merged**